### DISCUSSION TITLE
Is Bridgy Fed a PDS?

### DISCUSSION THREAD

**Discussion by singpolyma** [snarfed/bridgy-fed#1461] [General]
I'm curious about how this is implemented architecture-wise on the bluesky side.  Does Bridgy Fed effectively act as a PDS and then the network gets the content that way, or is there another approach being taken?

**Comment by snarfed** [comment]
Hi @singpolyma! Good to see you here! Yes, BF is effectively a PDS. It stores repos and serves their contents over `subscribeRepos`, `getBlocks`, `getRecord` etc to the Bluesky relay and anyone else, and it's in bridged users' DID docs in the `service` section, eg https://web.plc.directory/did/did:plc:2xgjhj6jge6kgj73szlasuno :

```json
  "service": [
    {
      "id": "#atproto_pds",
      "serviceEndpoint": "https://atproto.brid.gy",
      "type": "AtprotoPersonalDataServer"
    }
  ],
```

**Reply by singpolyma** [reply]
Nice! Using plc dids for now even for bridged websites I guess? Path of least resistance less setup for users.

**Reply by Tamschi** [reply]
It also (theoretically, eventually) should allow keeping the account if the domain has to be replaced with another for some reason.
